src/gui/tray/syncstatussummary.cpp

```diff
@@ -207,6 +216,12 @@ void SyncStatusSummary::onFolderProgressInfo(const ProgressInfo &progress)
     const qint64 totalSize = qMax(completedSize, progress.totalSize());
     const qint64 totalFileCount = qMax(currentFile, progress.totalFiles());
 
+    if (_totalFiles <= 0 && totalFileCount > 0) {
+        setSyncStatusString(tr("Syncing"));
+    }
+
+    setTotalFiles(totalFileCount);
+
```

Comment:
This is a little confusing when reading, what is the difference between `_totalFiles` and `totalFileCount`? Would it be better to use more distinct variable names here?